src/utils/Processing.ts

```diff
@@ -199,8 +213,9 @@ export function writeFHIRResources(
     folder: string,
     resources: { getFileName: () => string; toJSON: (snapshot: boolean) => any }[]
   ) => {
-    folder = useGeneratedFolder ? 'generated' : folder;
-    const exportDir = path.join(outDir, 'input', folder);
+    const exportDir = useGeneratedFolder
```

Comment:
I think in this function, if the `useGeneratedFolder` flag is equivalent to the `isIgPubContext` flag in `app.ts`, can we just change the name here to match, so that within `writeFHIRResources`, the flag is called `isIgPubContext`?

Reply:
That was there when I got here. ;-)  That said, it used to be used in the new IG Publisher context _and_ the legacy flat context.  Now that it's used only in the new IG Publisher context, your suggestion makes total sense.